src/app/index/index.component.css

```diff
@@ -171,9 +171,9 @@ footer {
   }
 }
 
-@media screen and (max-width: 479px) {

```

Comment:
@harshit98 Please be extra careful when you change css, this was very painstakingly tested and written to ensure mobile responsiveness. Please when you make changes here
Before your change: 
![image](https://user-images.githubusercontent.com/20185076/28992635-82fb84bc-79bf-11e7-8225-2616e83a4480.png)

After your change:
![image](https://user-images.githubusercontent.com/20185076/28992629-6a6f9438-79bf-11e7-9875-1f366da35dd6.png)